stable/locust/values.yaml

```diff
@@ -102,6 +102,7 @@ ingress:
   #    hosts:
   #      - chart-example.local
 
+configMaps: {}
```

Comment:
Can we rename this just so it's distinct from the other configmap values? Let's also add a doc item so that readme is updated

```suggestion
#  extraConfigMaps -- Any extra configmaps to mount for the master and worker. Can be used for extra python packages
extraConfigMaps: {}
```